k8s_handle/__init__.py

```diff
@@ -15,19 +15,125 @@
 from k8s_handle.k8s.deprecation_checker import ApiDeprecationChecker, DeprecationError
 from k8s_handle.k8s.resource import Provisioner, ProvisioningError
 
+COMMAND_DEPLOY = 'deploy'
+COMMAND_DESTROY = 'destroy'
+
 log = logging.getLogger(__name__)
 logging.basicConfig(level=settings.LOG_LEVEL, format=settings.LOG_FORMAT, datefmt=settings.LOG_DATE_FORMAT)
 
+
```

Comment:
Thought about moving handlers into other file, but decided not to do this, "good enough".
Assigning subcommands handlers, DRY generalization.